source/Halibut.Tests/Support/BackwardsCompatibility/ProxyHalibutTestBinaryRunner.cs

```diff
@@ -123,15 +124,13 @@ void ProcessLogs(string s)
                         if (s.Contains("RunningAndReady")) hasTentacleStarted.Set();
                     }
 
-                    ShellExecutor.ExecuteCommand(new HalibutTestBinaryPath().BinPath(version),
-                        "",
-                        tmp.FullPath,
-                        ProcessLogs,
-                        ProcessLogs,
-                        ProcessLogs,
-                        customEnvironmentVariables: settings,
-                        cancel: cancellationToken
-                    );
+                    await Cli.Wrap(new HalibutTestBinaryPath().BinPath(version))
```

Comment:
ShellExecutor.ExecuteCommand routes debug, info, and error.

Cli.Wrap does output and error. 

Just wondering, does 'output' capture 'debug' (and info)?

Reply:
I think it lied, processes have only STDOUT and STDERR